Pear/View/FeedbackView.swift

```diff
@@ -0,0 +1,117 @@
+        arrowView.layer.insertSublayer(shape, at: 0)
+        arrowBackgroundView.addSubview(arrowView)
+
+        feedbackBackgroundView.backgroundColor = .white
+        addSubview(feedbackBackgroundView)
+
+        feedbackTableView.backgroundColor = .backgroundWhite
+        feedbackTableView.separatorStyle = .none
+        feedbackTableView.showsVerticalScrollIndicator = false
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.isScrollEnabled = false
+        feedbackTableView.register(FeedbackMenuTableViewCell.self, forCellReuseIdentifier: FeedbackMenuTableViewCell.reuseIdentifier)
+        feedbackBackgroundView.addSubview(feedbackTableView)
+    }
+
+    private func setupConstraints() {
+        arrowBackgroundView.snp.makeConstraints { make in
+            make.leading.trailing.top.equalToSuperview()
+            make.height.equalTo(size/2)
+        }
+        arrowView.snp.makeConstraints { make in
+            make.trailing.equalTo(arrowBackgroundView.snp.trailing)
+            make.bottom.equalTo(arrowBackgroundView.snp.bottom)
+            make.width.equalTo(size)
+            make.height.equalTo(size)
+        }
+        feedbackBackgroundView.snp.makeConstraints { make in
+            make.top.equalTo(arrowBackgroundView.snp.bottom)
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+        feedbackTableView.snp.makeConstraints { make in
+            make.leading.trailing.top.bottom.equalTo(feedbackBackgroundView)
+        }
+    }
+    
+}
+
+extension FeedbackView: UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        feedbackOptions.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = feedbackTableView.dequeueReusableCell(withIdentifier: FeedbackMenuTableViewCell.reuseIdentifier, for: indexPath) as? FeedbackMenuTableViewCell else { return UITableViewCell() }
+        let feedbackOption = feedbackOptions[indexPath.row]
+        cell.configure(for: feedbackOption)
+        return cell
+    }
+
+}
+
+extension FeedbackView: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        40
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let optionSelected = feedbackOptions[indexPath.row]
+        if optionSelected == "Send feedback" {
+            let feedbackURL = Keys.feedbackURL
+            if let url = URL(string: feedbackURL), UIApplication.shared.canOpenURL(url) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            }
+        } else {
+            let email = "team@cornellappdev.com"
+            let emailSubject = optionSelected == "Contact us" ? "Pear%20Feedback" : "Report%20User"
```

Comment:
`let emailSubject = optionSelected == "Contact us" ? "Pear Feedback" : "Report User"` is fine?